src/map/status.c

```diff
@@ -12319,6 +12326,7 @@ void status_defaults(void) {
 	status->calc_mob_ = status_calc_mob_;
 	status->calc_pet_ = status_calc_pet_;
 	status->calc_pc_ = status_calc_pc_;
+	status->calc_pc_additional = status_calc_pc_additional;
```

Comment:
Ah, you're right o.o the hook stub would...crash when it calls the original function?
We could modify it so that it always check the function pointer before calling it, if it's worth it.